src/test/java/seedu/address/model/person/passenger/PriceContainsKeywordsPredicateTest.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.person.passenger;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PRICE_AMY;
+import static seedu.address.logic.commands.CommandTestUtil.VALID_PRICE_BOB;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.testutil.PassengerBuilder;
+
+public class PriceContainsKeywordsPredicateTest {
```

Comment:
Might be a little late as this is the test code and not the actual class name but maybe we should rename this to something a bit more descriptive? Something like PriceIsLargerThanAmountPredicate? As right now it sounds like the predicate is checking if it contains a number instead of what its actually doing.

Reply:
Thanks for your comment! I will make the name more descriptive as suggested